release/mk-release-snapshots.sh

```diff
@@ -9,23 +9,48 @@
 SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
 
 # Hostname of the CI server where snapshots are built
-SQUID_BUILD_SERVER=buildmaster.squid-cache.org
+SQUID_BUILD_SERVER=build.squid-cache.org
 
 # file Jenkins uses to assemble the list of snapshot files
 outfile=HEAD.out
 
+# arguments:
+# jenkins job name
+# destination directory
+get_artifacts() {
+	local job=$1
+	local outdir=$2
+	local artifactsUrl="https://$SQUID_BUILD_SERVER/job/${job}/lastSuccessfulBuild/artifact/artifacts/*zip*/artifacts.zip"
+	rm artifacts.zip 2>&1 >/dev/null
+	if ! wget --quiet --ca-cert=/etc/ssl/certs/ISRG_Root_X1.pem "$artifactsUrl" ; then
+		echo "could not download artifacts from $artifactsUrl"
+		return 1
+	fi
+ 	rmdir -f artifacts 2>&1 >/dev/null
+	if ! unzip -qq artifacts.zip ; then
+		echo "could not extract files from ${job}:artifacts.zip"
+		rmdir -f artifacts 2>&1 >/dev/null
+  		return 1
+ 	fi
+	if ! test -d artifacts ; then
+		echo "${job}:artifacts.zip does not contain a subdirectory"
+		rm -d `unzip -Z1 artifacts.zip`
```

Comment:
I do not think this will work reliably (or at all) because `rm -d` does not remove non-empty directories while `unzip -Z1` may list directories before listing files inside those directories. For example:

```sh
$ unzip -Z1 t.zip 
a/
a/f2
a/f1
z/
z/f3
```

Writing `rm -rf` instead of `rm -d` would work AFAICT, but that is a bit scary since we are obviously in a spot where we should not trust what names the archive does contain (and whether all of them were actually extracted and, hence, can now be safely deleted).

Reply:
Debian/Ubuntu `unzip` does not seem to have this issue with our `artifacts.zip` files. So leaving as-is for now.